tests/Rule/LatteTemplatesRule/SimpleControl/Fixtures/Resolve/SomeControl.php

```diff
@@ -205,4 +206,10 @@ private function neverReturn(): void
     {
         die();
     }
+
+    public function renderRenderEngine(): void
```

Comment:
Will this work when it will be in some custom Class and not in Control / Presenter?

Reply:
It would as long as that custom class/method is matched by some TemplateResolver